### ISSUE TITLE
No Data Is Shown.

### ISSUE THREAD

**Issue by enigmaharjan** [CodelightStudios/Android-Smart-Login#48]
While logging in using LoginType.Facebook, it doesn't show any data except userid and gender. I think its time to update the library.

**Comment by kalyandechiraju** [comment]
Yeah. It's been a while I have updated this library and I believe it's using an older version of the SDK.
